lib/stdlib/doc/src/binary.xml

```diff
@@ -256,6 +256,25 @@
       </desc>
     </func>
 
+    <func>
+      <name name="encode_hex" arity="2" since="OTP 26.0"/>
+      <fsummary>Encode a binary into a hex encoded binary with specified case</fsummary>
+      <desc>
+          <p>Encode a binary into a hex encoded binary with specified case.
+          If the case is <c>lowercase</c>, the <c>a</c> through <c>f</c> in hex encoded binary
+          will be in lower case. If the case is <c>uppercase</c>, the <c>A</c> through <c>F</c>
+          in hex encoded binary will be in upper case.</p>
```

Comment:
```suggestion
      <fsummary>Encodes a binary into a hex encoded binary with specified case</fsummary>
      <desc>
          <p>Encodes a binary into a hex encoded binary using the specified case for the
          Hex digits "a" to "f".</p>
```
I think this is clear enough, and the examples below illustrate what will happen.

Reply:
Thank you for your advice.